test/controllers/questions_controller_test.rb

```diff
@@ -28,27 +36,23 @@ class AuthenticatedTest < ActionDispatch::IntegrationTest
 
       get api_questions_url, headers: { 'Authorization' => api_key }
 
-      assert_response :success
-      assert expected_response == response.body, "\nExpected:\n#{expected_response} \n\nActual:\n#{response.body.to_json}"
+      expect_successful_response(expected_response)
     end
 
     test 'counts api_key utilization' do
       tenant = create :tenant
```

Comment:
I actually like that you created the tennant in each test, rather than a before block.

Reply:
May I ask why? Usually people like the before/setup block.

Reply:
Depends how many tests are in the context. I almost extracted it but then I thought - why? You've already done the right thing, this is something important to the test, it's quick to skim, and it only takes an extra line per test 🤷🏽